buildSrc/src/main/kotlin/buildsrc/conventions/publishing.gradle.kts

```
@@ -63,6 +63,18 @@ publishing {
                     url.convention("https://opensource.org/licenses/Apache-2.0")
                 }
             }
+
+            developers {
+                developer {
+                    email.convention("adam@adamko.dev")
```

```suggestion
                    email.set("adam@adamko.dev")
```

nitpick, for consistency with the other developer block